src/components/Dashboard/Analytics/AnalyticsStackedBar.jsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import React from "react";
+import HSBar from "react-horizontal-stacked-bar-chart";
+import { HorizontalStackedbar } from "./AnalyticsStackedBarElements";
+import { horizontalStackedBardata } from "./AnalyticsUtils";
+
+export default function HorizontalStackedBar() {
+    return (
+        <HorizontalStackedbar>
```

Comment:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/55c4c438-7618-4f54-aa12-d99e6baa135d)

I think it will look better to write the 35% in the right side of the bar.

Reply:
I tried placing 35% at the right side but it didn't look beautiful to me.